### PR TITLE
Pxweb2 304 snappy scrolling and debounced search

### PR DESCRIPTION
**Pull request by michaelpande**
Can now switfly render and efficiently scroll through a seemingly infinite list of items in the variablebox. 
This is not a perfect implementation, as it does make keyboard navigation a bit worse, but that should probably be a seperate task otherwise this one would be too big.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,6 +30,9 @@
     "react-dom": "18.3.1",
     "react-i18next": "^15.0.1",
     "react-router-dom": "^6.26.2",
+    "react-virtualized": "^9.22.5",
```
Comment (michaelpande) [inline package.json:33]: Kan fjernes

File: package.json [hunk 2]
```diff
@@ -29,6 +30,9 @@
...
+    "react-virtualized-auto-sizer": "^1.0.24",
```
Comment (michaelpande) [inline package.json:34]: Kan fjernes

File: libs/pxweb2-ui/src/lib/components/VariableBox/VariableBoxContent/VariableBoxContent.tsx [hunk 3]
```diff
@@ -247,107 +294,64 @@ export function VariableBoxContent({
...
-                value={allValuesSelected}
-                onChange={() => onChangeMixedCheckbox(varId, allValuesSelected)}
-                ariaControls={valuesToRender.map((value) => value.code)}
-                strong={true}
-                inVariableBox={true}
+            {' '}
+            {items.length > 0 && (
+              <Virtuoso
+                computeItemKey={(key) => `item-${key}`}
+                ref={listRef}
```
Comment (michaelpande) [inline libs/pxweb2-ui/src/lib/components/VariableBox/VariableBoxContent/VariableBoxContent.tsx:327]: Se om denne kan fjernes

File: libs/pxweb2-ui/src/lib/components/VariableBox/VariableBoxContent/VariableBoxContent.tsx [hunk 4]
```diff
@@ -247,107 +294,64 @@ export function VariableBoxContent({
...
+                style={{ height: '380px', maxHeight: '380px', width: '100%' }}
```
Comment (michaelpande) [inline libs/pxweb2-ui/src/lib/components/VariableBox/VariableBoxContent/VariableBoxContent.tsx:328]: Dette kan være en klasse

File: libs/pxweb2-ui/src/lib/components/VariableBox/VariableBoxContent/VariableBoxContent.tsx [hunk 5]
```diff
@@ -247,107 +294,64 @@ export function VariableBoxContent({
...
+                className=""
+                totalCount={items.length}
+                itemContent={(index) => itemRenderer(items, index)}
+                scrollSeekConfiguration={{
+                  enter: (velocity) => Math.abs(velocity) > 1000,
+                  exit: (velocity) => Math.abs(velocity) < 30,
+                }}
+                components={{
+                  ScrollSeekPlaceholder: ({ height }) => (
+                    <div
```
Comment (michaelpande) [inline libs/pxweb2-ui/src/lib/components/VariableBox/VariableBoxContent/VariableBoxContent.tsx:338]: Dette skal bli en skeleton

**Review by MikaelNordberg — APPROVED** [review]
Looks good

**Pull request merged**